liftoff_giftlist/src/main/resources/templates/add_member.html

```diff
@@ -11,7 +11,8 @@
             <label th:for="username">Username
                 <input class="form-control" th:name="username">
             </label>
-            <p class="error"></p>
+            <p th:if="${!usernameNotFound}" class="error">username not found</p>
```

Comment:
My logic might be backwards, but aren't these conditionals supposed to work *without* the exclamation mark? Just double check that.